appshell/ios/Podfile

```diff
@@ -0,0 +1,10 @@
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '8.0'
+
+target 'AppShell' do
+    pod 'SDWebImage', '~>4'
+    pod 'SocketRocket', '~>0.5'
+    pod 'WeexSDK', '>=0.15'
```

Comment:
here may be the   pod 'WeexSDK', '~>0.15',  so it will get the latest version of SDK

Reply:
Thanks for the comment, I've updated the codes.
Anyway '>=0.15' will make sure the version number is no less than 0.15, which also helps the project to get the latest WeexSDK.